x/feegrant/types/filtered_fee.go

```diff
@@ -0,0 +1,120 @@
+package types
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	proto "github.com/gogo/protobuf/proto"
+)
+
+const (
+	AllowedMsgGas = 1000
```

Comment:
@robert-zaremba I'm wondering if you have any thoughts on how much gas we should charge here?